score/audit-checklist.md

```diff
@@ -132,19 +159,23 @@ Therefore, not only random function, but any attempt to prevent block generation
 won = datetime.datetime.now() % 2 == 0
 ```
 
-### Outbound network call
-
-Outbound network call is prohibited.  
-Outcome of network call from each node can be different.
-
+### Fixed SCORE Infomation
+SCORE's critical information can not be changed since it was first installed. In case of IRC2 token SCORE, name, sysmbol and decimals should not be changed; for other SCORE, name must not be changed.
```

Comment:
```suggestion
SCORE's critical information should not be changed once it has been deployed. In case of IRC2 token SCORE, `name`, `symbol` and `decimals` should not be changed; for other SCOREs, `name` must not be changed.

IRC token type must be fixed once deployed as well. For example, IRC2 token SCORE should not be updated to IRC3 tokens, and IRC2 token should not be updated to non-IRC SCORE. 
```